aws-athena/views/default-vw_pin_sale.sql

```diff
@@ -287,7 +310,13 @@ SELECT
     mydec_sales.is_homestead_exemption,
     mydec_sales.homestead_exemption_general_alternative,
     mydec_sales.homestead_exemption_senior_citizens,
-    mydec_sales.homestead_exemption_senior_citizens_assessment_freeze
+    mydec_sales.homestead_exemption_senior_citizens_assessment_freeze,
+    sales_val.sv_is_outlier,
+    sales_val.sv_is_ptax_outlier,
+    sales_val.sv_is_heuristic_outlier,
+    sales_val.sv_outlier_type,
+    sales_val.sv_run_id,
+    sales_val.sv_version
```

Comment:
This will give you the same thing and be more concise (as long as SQLfluff will allow it).

```suggestion
    sales_val.*
```

Reply:
Did not know this - nice clean up. Change has been made.